deployment/cdk/opensearch-service-migration/lib/dashboards/capture-replay-dashboard.ts

```diff
@@ -0,0 +1,117 @@
+import { CfnDashboard } from 'aws-cdk-lib/aws-cloudwatch';
+import { Construct } from 'constructs';
+import * as fs from 'fs';
+import * as path from 'path';
+
+export interface CaptureReplayDashboardProps {
+  readonly stage: string;
+}
+
+export class CaptureReplayDashboard extends Construct {
+  public readonly dashboard: CfnDashboard;
+
+  constructor(scope: Construct, id: string, props: CaptureReplayDashboardProps) {
+    super(scope, id);
+
+    // Read the dashboard JSON from file
+    const dashboardJsonPath = path.join(__dirname, 'migrationassistant-capture-replay-dashboard.json');
+    let dashboardJson;
+    
+    try {
+      const fileContent = fs.readFileSync(dashboardJsonPath, 'utf8');
+      dashboardJson = JSON.parse(fileContent);
+      
+      // Inject stage parameter into all metrics
+      this.injectStageParameter(dashboardJson, props.stage);
+      
+    } catch (error) {
+      // Fallback to default dashboard if JSON file is missing or invalid
+      console.warn(`Failed to load dashboard JSON from ${dashboardJsonPath}: ${error}. Using default dashboard.`);
+      dashboardJson = this.createDefaultDashboard(props.stage);
+    }
+
+    // Create the dashboard
+    this.dashboard = new CfnDashboard(this, 'Dashboard', {
+      dashboardName: `CaptureReplay-Dashboard-${props.stage}`,
+      dashboardBody: JSON.stringify(dashboardJson)
+    });
+  }
+
+  // Injects the stage parameter into all metrics in the dashboard JSON
+  private injectStageParameter(dashboardJson: any, stage: string): void {
```

Comment:
Check the reindex-from-snapshot dashboard, this shouldn't be necessary and we should just be able to set the STAGE variable and use it throughout the json file